.atomist/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Actually, I think we want

```javascript
"interface-name": [false, "always-prefix"]
```

Since we don't want it to complain if the prefix happens to start with `I`. The go linter complains about such things even when it is legit.